src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SparseTest.java

```diff
@@ -0,0 +1,57 @@
+package com.microsoft.sqlserver.jdbc.datatypes;
+
+import java.sql.*;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+import static org.junit.Assert.fail;
+
+import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.TestResource;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+
+@RunWith(JUnitPlatform.class)
+public class SparseTest extends AbstractTest {
+    final static String tableName = RandomUtil.getIdentifier("SparseTestTable");
+    final static String escapedTableName = AbstractSQLGenerator.escapeIdentifier(tableName);
+
+    @Test
+    public void testSparse() throws Exception {
+        try (Connection conn = DriverManager.getConnection(connectionString)) {
+            assumeTrue(!TestUtils.isSqlAzureDW(conn), TestResource.getResource("R_skipAzure"));
+            try (Statement stmt = conn.createStatement()) {
+
+                // Create the test table
+                TestUtils.dropTableIfExists(escapedTableName, stmt);
+
+                StringBuilder bd = new StringBuilder();
+                bd.append("create table " + escapedTableName + " (col1 int, col2 varbinary(max)");
+                for (int i = 3; i <= 1024; i++) {
+                    bd.append(", col" + i + " varchar(20) SPARSE NULL");
+                }
+                bd.append(")");
+                String query = bd.toString();
+
+                stmt.executeUpdate(query);
+
+                stmt.executeUpdate("insert into " + escapedTableName + " (col1, col2, col1023)values(1, 0x45, 'yo')");

```

Comment:
This test doesn't really make a lot of sense. Why did it create 1024 columns if it's just going to use one of them and leave the rest as "NULL". It also only verifies with col1023, and not col1 or col2. Can we clean this up a bit.